sdk/eventhub/event-hubs/README.md

```diff
@@ -117,69 +116,53 @@ main();
 
 ### Publish events to an Event Hub
 
-In order to publish events, you'll need to create an `EventHubProducer`. Producers may be dedicated to a specific partition, or allow the Event Hubs service to decide which partition events should be published to. It is recommended to use automatic routing when the publishing of events needs to be highly available or when event data should be distributed evenly among the partitions. In the below examples, we will take advantage of automatic routing.
-
-#### Send a single event or an array of events
-
-Use the [send](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubproducer.html#send) method to send a single event or multiple events using a single call.
-
-```javascript
-const { EventHubClient } = require("@azure/event-hubs");
-
-async function main() {
-  const client = new EventHubClient("connectionString", "eventHubName");
-  const producer = client.createProducer();
-
-  await producer.send({ body: "my-event-body" });
-  await producer.send([{ body: "foo" }, { body: "bar" }]);
-
-  await producer.close();
-  await client.close();
-}
-
-main();
-```
+In order to publish events, you'll need to create an `EventHubProducerClient`. Producers may be dedicated to a specific partition, or allow the Event Hubs service to decide which partition events should be published to. It is recommended to use automatic routing when the publishing of events needs to be highly available or when event data should be distributed evenly among the partitions. In the examples below, we will take advantage of automatic routing.
 
 #### Send a batch of events
 
 Use the [createBatch](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubproducer.html#createbatch) method to create
-an `EventDataBatch` object which can then be sent using the [send](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubproducer.html#send) method.
+an `EventDataBatch` object which can then be sent using the [sendBatch](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubproducerclient.html#sendbatch) method.
 Events may be added to the `EventDataBatch` using the [tryAdd](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventdatabatch.html#tryadd)
 method until the maximum batch size limit in bytes has been reached.
 
 ```javascript
 const { EventHubClient } = require("@azure/event-hubs");
```

Comment:
```suggestion
const { EventHubProducerClient } = require("@azure/event-hubs");
```

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/fa55002b8df0ed60a4a3cb6fa5c258afe30a3d7c